common/src/test/scala/com/nvidia/spark/rapids/ThreadFactoryBuilderTest.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids
+
+import java.util.concurrent.{Callable, Executors, ExecutorService}
+
+import org.scalatest.FunSuite
+
+class ThreadFactoryBuilderTest extends FunSuite {
+
+  test("test thread factory builder") {
+    var pool1: ExecutorService = null
```

Comment:
Nit: There's no need for `var` or `null` here.  This should simply be:
```scala
val pool1 = Executors.newFixedThreadPool(...)
try {
  ...
} finally {
  pool1.shutdown()
}
```
Applies to the other instances of this.